cpp/test/Slice/errorDetection/NotType.ice

```diff
@@ -24,7 +24,7 @@ module Test
 
     interface I
     {
-        E foo(E e1; E e2);
+        E foo(E e1, E e2);
```

Comment:
Using a `;` here is completely bogus.
But we weren't reporting a syntax error, because `E` isn't a valid type.

Now the compiler correctly reported the `;` and I fixed it.